test/integration-override/handlers/positions/handlerBatch.test.js

```diff
@@ -1838,6 +1839,10 @@ Test('Handlers test', async handlersTest => {
       await testConsumer.destroy() // this disconnects the consumers
 
       await Producer.disconnect()
+      try {
+        await ProxyCache.proxyCache.disconnect()
+      } catch (err) {
+      }
```

Comment:
maybe, we can add, at least, log with error here? (for easier debugging in case of error)

Reply:
Steven refactored this in a follow up PR.